src/output.c

```diff
@@ -880,6 +881,49 @@ output_nearest_to_cursor(struct server *server)
 		server->seat.cursor->y);
 }
 
+struct output *
+output_get_adjacent(struct output *output, enum view_edge edge, bool wrap)
+{
+	assert(output);
```

Comment:
If we let output be NULL, we can use `output_get_adjacent` as a full replacement for `view_get_adjacent_output` and just kill that function entirely:
```suggestion
	if (!output) {
		output = output_nearest_to_cursor(server);
	}
	
	if (!output_is_usable(output)) {
		wlr_log(WLR_ERROR,
			"output is not usable, cannot find adjacent output");
		return NULL;
	}
```